feeds/wlan-ap/opensync/src/platform/openwrt/src/command/src/cmd_portforwarding.c

```diff
@@ -18,6 +21,31 @@ static void cmd_handler_port_forwarding_cb(struct ev_loop *loop, ev_child *child
 	task_status(task, TASK_COMPLETE, NULL);
 }
 
+int cmd_handler_stop_session(struct task *task)
+{
+	pid_t pid = 0;
+	char buf[BUF_SIZE];
+
+	FILE *cmd_pipe = popen("pidof portfwd", "r");
+
+	if (cmd_pipe) {
+	fgets(buf, BUF_SIZE, cmd_pipe);
+	pid = strtoul(buf, NULL, 10);
+	}
+
```

Comment:
pclose() needs to be inside the if (cmd_ppipe)

Reply:
done